documentation/src/main/asciidoc/topics/ref_transcoders.adoc

```diff
@@ -0,0 +1,10 @@
+[id='transcoders_{context}']
+= Transcoders
+Transcoders convert entries to and from different media types.
```

Comment:
this section is lacking everything... ok, I know what is a Transcoder but where is it configured? how to use it? when to use it? 
looking at the code it seems more like a private interface.

Reply:
@pruivo yeah I think this section needs some more love. one of the issues that we discussed in the dogfooding session that lead to this PR is that the transcoding content in the docs is advanced stuff, but it's currently mixed in with other details about encoding. 

I've moved this content to the end of the title to keep it kind of separate and available for "power users" but it still needs to be expanded and improved. that is in the content plan for Infinispan 13 but out of scope for this PR.

Reply:
@pruivo Do you think my earlier comment ^ is OK? I don't want to remove/drop the content about Transcoders but I don't think we should try and address all the concerns in this PR because it'd never get merged.

Reply:
I would remove it. 
The only way to register transcoder is via internal API which we don't have any docs for It. it does not make sense to add docs to something that users cannot do.

Reply:
ok. I'll comment this topic so it doesn't show up in the rendered documentation but we'll still have the basic details as a starting point for the next iteration.